protocols/identify/src/behaviour.rs

```diff
@@ -202,6 +202,41 @@ impl Config {
         self.hide_listen_addrs = b;
         self
     }
+
+    /// Get the protocol version of the Config.
+    pub fn protocol_version(&self) -> String {
```

Comment:
Return `&str` to avoid potential unnecessary cloning. The end user will clone if he needs to.

Reply:
Done.